### DISCUSSION TITLE
NetworkError: Failed to execute 'send' on 'XMLHttpRequest': Failed to load

### DISCUSSION THREAD

**Discussion by krseguros** [YetiForceCompany/YetiForceCRM#17296] [How to]
Hello,

YetiForce CRM 6.4 Service Pack v6

When assigning a task, also when postponing, also when completing from the notification window and from list view it hangs and you have to reload the page to continue working.

What could be causing this inappropriate behavior?

When assigning a record
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/96985476/59ce3d78-0200-4cdb-9e04-785bc4dcf8ee)
When postponing or completing
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/96985476/60672e62-57d9-4fb1-9bf6-d1b8005e0386)
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/96985476/69f9e355-5095-415d-bd76-de166702a6ae)

**Comment by mariuszkrzaczkowski** [comment]
logs?

**Reply by krseguros** [reply]
Thanks @mariuszkrzaczkowski in that case I will have to seek help from a third party to verify this because I don't know how to create or generate the logs

**Reply by rskrzypczak** [reply]
https://doc.yetiforce.com/developer-guides/debug/
